setup.cfg

```diff
@@ -3,3 +3,15 @@ universal=1
 
 [aliases]
 test=pytest
+
+[flake8]
+exclude=datadotworld/client/_swagger/*, build, .eggs, .tox/*
+ignore=F401,E121,F403,F405,F841
```

Comment:
Instead of ignoring these warnings globally, I would rather address each instance in code with `# noqa`